src/test/java/com/algorand/algosdk/unit/Rekeying.java

```diff
@@ -0,0 +1,75 @@
+package com.algorand.algosdk.unit;
+
+import java.security.GeneralSecurityException;
+import java.security.NoSuchAlgorithmException;
+
+import com.algorand.algosdk.account.Account;
+import com.algorand.algosdk.builder.transaction.PaymentTransactionBuilder;
+import com.algorand.algosdk.builder.transaction.TransactionBuilder;
+import com.algorand.algosdk.transaction.SignedTransaction;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class Rekeying {
```

Comment:
If you include `Stepdefs` in this file, do you still need to re-implement these?
```
src/test/java/com/algorand/algosdk/integration/Stepdefs.java
243:    @Given("payment transaction parameters {int} {int} {int} {string} {string} {string} {int} {string} {string}")

src/test/java/com/algorand/algosdk/integration/Stepdefs.java
281:    @Given("mnemonic for private key {string}")

src/test/java/com/algorand/algosdk/integration/Stepdefs.java
900:    @When("I create the flat fee payment transaction")

src/test/java/com/algorand/algosdk/integration/Stepdefs.java
353:    @When("I sign the transaction with the private key")

src/test/java/com/algorand/algosdk/integration/Stepdefs.java
358:    @Then("the signed transaction should equal the golden {string}")
```

Reply:
Maybe there is a way, but cucumber was looking for those implementations here.